docs/source/markdown/options/mount.md

```diff
@@ -26,10 +28,12 @@ Current supported mount TYPEs are **bind**, **volume**, **image**, **tmpfs** and
 
        Common Options:
 
-	      · src, source: mount source spec for bind and volume. Mandatory for bind.
+	      · src, source: mount source spec for bind, glob, and volume. Mandatory for bind and glob.
 
 	      · dst, destination, target: mount destination spec.
 
+	      Paths matching globs, are mounted on the destination directory with the identical name inside the container.
```

Comment:
Extraneous comma, please remove. And, would it make sense to say "with the identical basename" or "identical filename"? Someone could possibly interpret that as "/foo/bar* will get mounted as /dest/foo/bar1, /dest/foo/bar2"